tests/integration/profiles_management/helpers/k8s.py

```diff
@@ -0,0 +1,146 @@
+import logging
+from pathlib import Path
+from typing import List
+
+import pytest
+import tenacity
+from lightkube import Client, codecs
+from lightkube.core.exceptions import ApiError
+from lightkube.generic_resource import GenericGlobalResource, GenericNamespacedResource
+from lightkube.resources.core_v1 import Namespace
+
+# silence default INFO logs of httpx, to avoid seeing
+# a log line for every request that happens with that module
+logging.getLogger("httpx").setLevel(logging.WARNING)
+log = logging.getLogger(__name__)
+
+
+# For errors when a Namespace exists while it shouldn't
+class ObjectStillExistsError(Exception):
+    """Exception for when a K8s object exists, while it should have been removed."""
+
+    pass
+
+
+def get_name(res: GenericNamespacedResource | GenericGlobalResource) -> str:
```

Comment:
If I understand correctly, we are duplicating the functions `get_name` and `get_namespace` that already exist under `src/profiles_management/helpers/k8s.py`, we could use the function from that module right?